files/vision.tex

```diff
@@ -0,0 +1,23 @@
+\chapter{Vision}
+This chapter will give insights into the vision this project is based upon and the boundaries for what this project will focus upon.
+
+
+\section*{The big picture}
+
+The vision behind this project is to revolutionize the club management and progress monitoring inside badminton clubs. The main part where this product will be different is a smart racket that can give the club, coach and individual player a bunch of information about their badminton performance. This information can help the coaches finding weaknesses and show them where to focus their training on a specific player. 
+
+The management portion would be a web application where management staff could handle business such as economy, practice attendance and events such as parties and game scheduling.
```

Comment:
I think 'portion' should be 'part'
As this is the vision, I think we can phrase it as 'is' instead of 'would', because this is what we want to do, but as you limit the scope later on, it is still clear it will not be done in this project, ?